kafka-observ-lib/main.libsonnet

```diff
@@ -30,7 +30,10 @@ local zookeeperlib = import 'zookeeper-observ-lib/main.libsonnet';
           uid: this.config.uid,
           dashboardNamePrefix: this.config.dashboardNamePrefix,
           dashboardTags: this.config.dashboardTags,
-          metricsSource: this.config.metricsSource,
+          metricsSource:
+            if this.config.metricsSource == 'prometheus' then 'prometheus'
+            else if this.config.metricsSource == 'grafanacloud' then 'grafanacloud'
+            else if this.config.metricsSource == 'bitnami' then 'prometheus',
```

Comment:
Just to double check, we want to force the metrics source = prom if it's a bitnami chart? (I assume that's the intent, but wanted to make sure it's not a copy-paste issue)

Reply:
yepp, that's for zookeeper's library  only: is it 'prometheus'  or grafanacloud there only.